ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/download/clientlibs/site/js/download.js

```diff
@@ -47,19 +48,41 @@ jQuery((function(ns, semanticModal, licenseModal, downloadService) {
                 id: DOWNLOAD_MODAL_ID,
                 url: DOWNLOAD_URL,
                 data: formData.serialize(),
-                options: {}
+                options: {
+                    closePrevious: function(htmlResponse, modalTracker) {
+                        var modal = $("<div>" + htmlResponse + "</div>").find(ns.Elements.selector(getId()));
+                        if($(modal).data(DOWNLOAD_DIRECT)) {
+                            for(var modalId of modalTracker) {
+                                if(modalId != licenseModal.id()) {
```

Comment:
```suggestion
                                if(modalId !== licenseModal.id()) {
```

.....cuz javascript!